burr/cli/__main__.py

```diff
@@ -175,10 +177,113 @@ def generate_demo_data():
         generate_all("burr/tracking/server/demo_data")
 
 
+def _transform_state_to_test_case(state: dict, action_name: str, test_name: str) -> dict:
+    """Helper function to transform a state into a test case.
+
+    :param state:
+    :param action_name:
+    :param test_name:
+    :return:
+    """
+    return {
+        "action": action_name,
+        "name": test_name,
+        "input_state": state,
+        "expected_state": {"TODO:": "fill this in"},
+    }
+
+
+@click.group()
+def test_case():
+    """Test case related commands."""
+    pass
+
+
+PYTEST_TEMPLATE = """import pytest
+from burr.testing import pytest_generate_tests  # noqa: F401
+# TODO: import action you're testing, i.e. import {ACTION_NAME}.
+
+@pytest.mark.file_name("{FILE_NAME}")
+def test_{ACTION_NAME}(input_state, expected_state):
+    \"\"\"Function for testing the action\"\"\"
+    input_state = state.State(input_state)
+    expected_state = state.State(expected_state)
+    _, output_state = {ACTION_NAME}(input_state)  # exercise the action
+    assert output_state == expected_state
+"""
+
+
+@click.command()
+@click.option("--project-name", required=True, help="Name of the project")
+@click.option("--partition-key", required=True, help="Partition key to look at")
+@click.option("--app-id", required=True, help="App ID to pull from.")
+@click.option("--sequence-id", required=True, help="Sequence ID to pull.")
+@click.option(
+    "--target-file-name",
+    required=False,
+    help="What file to write the data to. Else print to console.",
+)
+def create_test_case(
```

Comment:
We should note that this is `local` -- either as a subcommand or a parameter.

Also maybe add a `testing` file in the CLI and import it? This is getting a little unwieldy.

Reply:
what do you mean local?  Otherwise we should just refactor on the next addition.

Reply:
Sorry I mean it uses the `LocalTrackingClient`. E.G. namespace now then leave room for more later.